ocaml/xapi-consts/api_errors.ml

```diff
@@ -118,6 +118,7 @@ let pif_incompatible_primary_address_type = "PIF_INCOMPATIBLE_PRIMARY_ADDRESS_TY
 let required_pif_is_unplugged = "REQUIRED_PIF_IS_UNPLUGGED"
 let pif_not_present = "PIF_NOT_PRESENT"
 let pif_does_not_allow_unplug = "PIF_DOES_NOT_ALLOW_UNPLUG"
+let pif_in_fcoe_not_allow_unplug = "PIF_IN_FCOE_NOT_ALLOW_UNPLUG"
```

Comment:
I think you missed change in datamodel.ml.

Reply:
Yes, I will add this new-added error and the remaining errors to `datamodel.ml`